_book/02-first-steps/05-npm-usage.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Usage with npm"
+---
+
+The latest stable version is available via NPM registry. The version distributed via NPM it the WebAssembly build. It can be installed with: 
```

Comment:
```suggestion
The latest stable version is available in the NPM registry. The version distributed via NPM is the WebAssembly build. It can be installed with: 
```